pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -97,4 +99,24 @@ public Strategy getDefaultPartialUpsertStrategy() {
   public String getComparisonColumn() {
     return _comparisonColumn;
   }
+
+  public void setMode(Mode mode) {
+    _mode = mode;
+  }
+
+  public void setHashFunction(HashFunction hashFunction) {
+    _hashFunction = hashFunction;
+  }
+
+  public void setPartialUpsertStrategies(Map<String, Strategy> partialUpsertStrategies) {
```

Comment:
Can we add java doc here telling that key in this map should be columnName.

Reply:
Gotcha, let me add it.